packages/manager/modules/pci/src/projects/project/storages/databases/components/dashboard-integrations/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,6 @@
+{
+  "pci_databases_dashboard_integrations_title": "Intégrations des services",
+  "pci_databases_dashboard_integrations_integration_type": "{{type}} :",
```

Comment:
**nitpick:** Maintain as a constant instead.

Reply:
I'm not sure to understand what should be maintained as constant here. "{{type}} :" should be translated as the ponctuation rules are different between languages (space or not before the ":").